File: app/src/main/java/com/mtime/player/bean/MTimeVideoData.java
package com.mtime.player.bean;

import com.kk.taurus.playerbase.entity.DataSource;

/**
 * Created by mtime on 2017/10/19.
 */

public class MTimeVideoData extends DataSource {

    /**
     * 视频ID
     */
    private String videoId;
    /**
     * 视频来源
     */
    private int source;

    /**
     * 文件大小
     */
    private long fileSize;

    public long getFileSize() {
        return fileSize;
    }

    public void setFileSize(long fileSize) {
        this.fileSize = fileSize;
    }

    public MTimeVideoData() {
    }

    public MTimeVideoData(String videoId, int source) {
        this.videoId = videoId;
        this.source = source;
    }

    public MTimeVideoData(String data) {
        super(data);
    }

    public String getVideoId() {
        return videoId;
    }

    public void setVideoId(String videoId) {
        this.videoId = videoId;
    }

    public int getSource() {
        return source;
    }

    public void setSource(int source) {
        this.source = source;
    }
}

File: player/src/main/java/com/kotlin/android/player/bean/MTimeVideoData.kt
package com.kotlin.android.player.bean

import com.kk.taurus.playerbase.entity.DataSource

/**
 * @author ZhouSuQiang
 * @email zhousuqiang@126.com
 * @date 2020/7/15
 *
 * 时光视频实体
 */
class MTimeVideoData(
        data: String = "",
        var videoId: Long,
        var source: Long,
        var fileSize: Long = 0L
) : DataSource(data)